packages/netlify-cms-widget-markdown/src/MarkdownControl/schema.js

```diff
@@ -0,0 +1,310 @@
+import { Inline, Text } from 'slate';
```

Comment:
A proper schema - it isn't exactly exhaustive, but a great improvement over our previous setup.